wgengine/router/router_func.go

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2020 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package router
+
+// FuncRouter is a Router to the functions it embeds.
```

Comment:
add this line up here somewhere:

```go
var _ Router = FuncRouter{}
```

That'll make sure that FuncRouter implements Router (if we ever modify Router), since nothing else is this repo is currently assigning it to a Router.

Reply:
Moved into ipn-go-bridge.